theme/templates/resource-landing-page/title-section.html

```diff
@@ -51,6 +51,12 @@
                         <br>
                         <span>Content files are required to publish the resource or make the resource public or discoverable.</span>
                 {% endif %}
+                {%  if cm.resource_type|lower == "collectionresource" %}
+                    {% if not cm.has_resources%}
+                        <br>
+                        <span>Collection should contain at least one resource to be made it published, public or discoverable.</span>
+                    {% endif %}
+                {% endif %}
```

Comment:
@pkdash Agreed the current message is a bit confusing and your suggestion for the new message wording sounds good to me, but I suggest a more concise wording for your reference: "Metadata Status: all required metadata provided."